### ISSUE TITLE
Add support for WT32-ETH01

### ISSUE THREAD

**Issue by nullos007** [platformio/platform-espressif32#448]
Hi, it's possible to add this board : WT32-ETH01 ( with WT32-S1 MCU).

Thanks.

**Comment by SzczepanLeon** [comment]
Here is LAN8720 configuration for that board:

```
//GPIO16 = set to HIGH
//GPIO18 = ETH MDIO
//GPIO23 = ETH MDC
#define ETH_TYPE      ETH_PHY_LAN8720
#define ETH_CLKMODE   ETH_CLOCK_GPIO0_IN
#define ETH_ADDR      1
```

**Comment by efirshik** [comment]
where find work yaml config for this board?

**Comment by nullos007** [comment]
> 
> 
> Here is LAN8720 configuration for that board:
> 
> ```
> //GPIO16 = set to HIGH
> //GPIO18 = ETH MDIO
> //GPIO23 = ETH MDC
> #define ETH_TYPE      ETH_PHY_LAN8720
> #define ETH_CLKMODE   ETH_CLOCK_GPIO0_IN
> #define ETH_ADDR      1
> ```

Thanks for the reply 👍

**Comment by mondbaron** [comment]
solved by pull request #689

**Comment by valeros** [comment]
Added in [v4.2.0](https://github.com/platformio/platform-espressif32/releases/tag/v4.2.0)
